### PR TITLE
{chem}[intel-compilers/2022.1.0] NCIPLOT v4.2-20221021

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
@boegelbot please test @ jsc-zen2

**Review by smoors — APPROVED** [review]
lgtm

File: easybuild/easyconfigs/n/NCIPLOT/NCIPLOT-4.2-20221021-intel-compilers-2022.1.0.eb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+start_dir = 'src_%%(namelower)s_%s' % _formal_version
+
+prebuildopts = "sed -i 's/include Makefile.inc//;s/nciplot: $(OBJS) $(LIBS)/nciplot: $(OBJS)/g' Makefile && "
+prebuildopts += "make clean && "
+buildopts = 'LIBS="$LIBS"'
+
+# test scripts are still hardcoded for old version 4.0
+pretestopts = "cd %(builddir)s/%(namelower)s-*/tests && "
+pretestopts += "find . -name runtests.sh -exec sed -i 's/nciplot_4.0/nciplot_%s/g' {} \; && " % _formal_version
```
Comment (smoors) [inline easybuild/easyconfigs/n/NCIPLOT/NCIPLOT-4.2-20221021-intel-compilers-2022.1.0.eb:28]:
```suggestion
pretestopts += r"find . -name runtests.sh -exec sed -i 's/nciplot_4.0/nciplot_%s/g' {} \; && " % _formal_version
```

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen3

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Review by smoors — CHANGES_REQUESTED** [review]
lgtm

**Review by smoors — APPROVED** [review]

**Comment by smoors** [comment]
Going in, thanks @lexming!

**Pull request merged**